mlir/include/mlir-c/Dialect/EmitC.h

```diff
@@ -19,6 +19,117 @@ extern "C" {
 
 MLIR_DECLARE_CAPI_DIALECT_REGISTRATION(EmitC, emitc);
 
+enum MlirEmitCCmpPredicate : uint64_t {
+  MLIR_EMITC_CMP_PREDICATE_EQ = 0,
+  MLIR_EMITC_CMP_PREDICATE_NE = 1,
+  MLIR_EMITC_CMP_PREDICATE_LT = 2,
+  MLIR_EMITC_CMP_PREDICATE_LE = 3,
+  MLIR_EMITC_CMP_PREDICATE_GT = 4,
+  MLIR_EMITC_CMP_PREDICATE_GE = 5,
+  MLIR_EMITC_CMP_PREDICATE_THREE_WAY = 6,
+};
+
+//===---------------------------------------------------------------------===//
+// ArrayType
+//===---------------------------------------------------------------------===//
+
+MLIR_CAPI_EXPORTED bool mlirTypeIsAEmitCArrayType(MlirType type);
```

Comment:
```suggestion
MLIR_CAPI_EXPORTED bool mlirTypeIsAnEmitCArrayType(MlirType type);
```
with `An` instead of `A` or just
```suggestion
MLIR_CAPI_EXPORTED bool mlirTypeIsEmitCArrayType(MlirType type);
```
would be a bit nicer here and for all functions having `IsAEmitC` in their name, but this seems to follow the naming in https://mlir.llvm.org/docs/CAPI/#extensions-for-dialect-attributes-and-types.

Reply:
I though this was trying to express `isa<>` in C.

Reply:
Ah, you're probably right! Thanks :)